server/sonar-web/src/main/js/apps/background-tasks/components/TaskExecutionTime.js

```diff
@@ -23,11 +23,7 @@ import { formatDuration } from '../utils';
 /*:: import type { Task } from '../types'; */
 
 const TaskExecutionTime = ({ task } /*: { task: Task } */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove curly braces and "return" from this arrow function body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)